learn2learn/vision/datasets/mini_imagenet.py

```diff
@@ -92,18 +92,20 @@ def __init__(self, root, mode='train', transform=None, target_transform=None):
         f = open(pickle_file, 'rb')
         self.data = pickle.load(f)
 
-        self.x = torch.FloatTensor([np.transpose(x, (2, 0, 1)) for x in self.data['image_data']])
-        self.y = [-1 for _ in range(len(self.x))]
+        self.x = torch.from_numpy(self.data["image_data"]).permute(0, 3, 1, 2).float()
```

Comment:
fyi, from a preliminary test, This also improves the speed of dataset initialization